src/svgui/svgui.ts

```diff
@@ -250,6 +251,69 @@ export class Pair extends UIElement {
     }
 }
 
+/*
+ * Image for element.
+ * Backbone properties:
+ *     imageUrl: string
+ */
+export class Image extends UIElement {
+    public image: d3.Selection<any>;
+    public imageRect: d3.Selection<any>;
+
+    defaults() {
+        return _.extend(super.defaults(), {
+            imageUrl: undefined,
+            borderColor: 'green',
+            spacing: vector(10, 3),
+            padding: undefined,
```

Comment:
We can set as default padding (1px,1px,1px,1px) otherwise the image will be overlap the border line.